exp/addons/internal/webhooks/clusterresourcesetbinding_webhook.go

```diff
@@ -68,12 +82,13 @@ func (c *ClusterResourceSetBinding) validate(old *ClusterResourceSetBinding) err
 		)
 	}
 	var allErrs field.ErrorList
-	if old != nil && old.Spec.ClusterName != "" && old.Spec.ClusterName != c.Spec.ClusterName {
+	if oldCRSB != nil && newCRSB != nil && oldCRSB.Spec.ClusterName != "" && oldCRSB.Spec.ClusterName != newCRSB.Spec.ClusterName {
```

Comment:
```suggestion
	if oldCRSB != nil && oldCRSB.Spec.ClusterName != "" && oldCRSB.Spec.ClusterName != newCRSB.Spec.ClusterName {
```
I think it's fine to not check new. newCRSB is never nil